### PR TITLE
Replace Win32 DiskImager with Etcher

### PR DESCRIPTION
**Pull request by jviotti**
Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

**Comment by jviotti** [comment]
I'll reduce the resolution of some of the images.

File: START.htm [hunk 1]
```diff
@@ -177,17 +177,17 @@ <h2 id="update">Update board with latest software</h2>
...
+    <img src="Docs/images/install-etcher.png" width="50%" /><br />
     <p>Some general help on programming SD cards can be found on the <a target="_blank" href="http://help.ubuntu.com/community/Installation/FromImgFiles">Ubuntu Image Writer page</a>. 
 
     <h3>Step #5: Connect SD card to your computer</h3>
     <p>Use the provided microSD card to SD adapter or a USB adapter to connect the SD card to your computer.</p>
 
     <h3>Step #6: Write the image to your SD card</h3>
-    <p>Use either the Ubuntu Image Writer or instructions on its page to write the decompressed image to your SD card.</p>
-    <img src="Docs/images/write-latestimage.png" />
+    <p>Use Etcher to write the decompressed image to your SD card.</p>
```
Comment (lurch) [inline START.htm]: Etcher natively supports compressed images, so maybe we could take out the decompression instructions too?

File: START.htm [hunk 2]
```diff
@@ -177,17 +177,17 @@ <h2 id="update">Update board with latest software</h2>
     <img src="Docs/images/extract-latestimage.png" width="50%" /><br />
 
     <h3>Step #4: Install SD card programming utility</h3>
-    <p>Download and install <a target="_blank" href="https://launchpad.net/win32-image-writer/+download">Image Writer for Windows</a>. Be sure to download the binary distribution.</p>
-    <img src="Docs/images/download-win32diskimager.png" width="50%" /><br />
-    <img src="Docs/images/extract-win32diskimager.png" width="50%" /><br />
+    <p>Download and install <a target="_blank" href="https://etcher.io/">Etcher for Windows</a>.</p>
```
Comment (lurch) [inline START.htm]: Why restrict this to "Etcher for Windows"? We could just say "Download and install [Etcher](https://etcher.io/)"?

File: START.htm (hunk 1 above)
Comment (jviotti) [inline START.htm]: The images referenced in this tutorial are 7-zip compressed, which is not a format we currently support.

File: START.htm (hunk 2 above)
Comment (jviotti) [inline START.htm]: Makes sense, done!

File: START.htm (hunk 1 above)
Comment (lurch) [inline START.htm]: https://github.com/jadonk/beaglebone-getting-started/pull/30/files#diff-3e3d57b22232bb70d800b4f6bf126024R167 says "The file you download will have an .img.xz extention." ?

Comment (lurch) [inline START.htm]: (and if you wanted to I guess your could fix their typo to "extension" :grin: )

Comment (jviotti) [inline START.htm]: Oh, true. Why did I assume this was 7-zip then? -.- I'll completely remove the decompressiion instructions.

Comment (jviotti) [inline START.htm]: Done!

File: START.htm [hunk 3]
```diff
@@ -164,35 +164,26 @@ <h2 id="update">Update board with latest software</h2>
...
 
-    <h3>Step #5: Connect SD card to your computer</h3>
+    <h3>Step #3: Connect SD card to your computer</h3>
     <p>Use the provided microSD card to SD adapter or a USB adapter to connect the SD card to your computer.</p>
 
-    <h3>Step #6: Write the image to your SD card</h3>
-    <p>Use either the Ubuntu Image Writer or instructions on its page to write the decompressed image to your SD card.</p>
-    <img src="Docs/images/write-latestimage.png" />
+    <h3>Step #4: Write the image to your SD card</h3>
+    <p>Use Etcher to write image to your SD card.</p>
```
Comment (lurch) [inline START.htm]: "write image" -> "write the image"

File: START.htm [hunk 4]
```diff
@@ -164,35 +164,26 @@ <h2 id="update">Update board with latest software</h2>
...
 
-    <h3>Step #7: Eject the SD card</h3>
+    <h3>Step #5: Eject the SD card</h3>
     <p>Eject the newly programmed SD card.</p>
 
-    <h3>Step #8: Boot your board off of the SD card</h3>
+    <h3>Step #6: Boot your board off of the SD card</h3>
     <p>Insert SD card into your (powered-down) board, hold down the USER/BOOT button (if using Black) and apply power, either by the USB cable or 5V adapter.</p>
     <p>If using an original white BeagleBone, you are done.</p>
     <p>If using BeagleBone Black and the image is meant to program your on-board eMMC, you'll need to wait while the programming occurs.  When the flashing is complete, all 4 USRx LEDs will be lit solid.  <em>This can take up to 45 minutes.</em>  Power-down your board, remove the SD card and apply power again to be complete.</p>
```
Comment (lurch) [inline START.htm]: Unrelated to the changes you've made here, but I think "apply power again to finish." might sound better than "apply power again to be complete." ?

File: START.htm (hunk 3 above)
Comment (lurch) [inline START.htm]: And I wonder if it'd be worth clarifying (just to remove any possible confusion) that Etcher will transparently decompress the image on-the-fly before writing it to the SD card?

**Comment by lurch** [comment]
I think `install-etcher.png` would look nicer if you could take it again without the 'background text' showing through the semi-transparent title bar ;-)

File: START.htm (hunk 3 above)
Comment (jviotti) [inline START.htm]: Sounds good, done!

File: START.htm (hunk 4 above)
Comment (jviotti) [inline START.htm]: Probably worth sending another PR for that? What does @jadonk think?

**Comment by jviotti** [comment]
> I think install-etcher.png would look nicer if you could take it again without the 'background text' showing through the semi-transparent title bar ;-)

Done!

File: START.htm (hunk 4 above)
Comment (lurch) [inline START.htm]: Ahh, that's a good point ;-)   #31

**Pull request merged**